app/helpers/application_helper.rb

```diff
@@ -9,4 +9,15 @@ def alert_class_for(flash_type)
     }
     types[flash_type.to_sym]
   end
+
+  def all_months_select_options(first_month, last_month)
+    all_months(first_month, last_month).map { |month| [month.strftime('%Y年%m月'), month] }
+  end
+
+  def all_months(first_month, last_month)
+    loop.each_with_object([first_month]) do |_, days|
+      nm = days.last.next_month
```

Comment:
__[reek]__
* [FeatureEnvy] refers to 'days' more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)

<!-- Comment by [SideCI](https://sideci.com) -->